inc/admin.php

```diff
@@ -1058,4 +1062,42 @@ private function get_dashboard_strings() {
 		];
 	}
 
+	/**
+	 * Allow SVG uploads
+	 *
+	 * @param array $mimes Supported mimes.
+	 *
+	 * @return array
+	 * @access public
+	 */
+	public function allow_meme_types( $mimes ) {
```

Comment:
Lets also add the filter that the function is attached to like 8) from here https://github.com/Codeinwp/dev-internals/blob/main/PlatformOps/Optimole/README.md .